### PR TITLE
Add survey configuration

### PR DESCRIPTION
**Pull request by rasmustautsglia**
Add survey customization through remote configuration

MOB-1555

### CODE REVIEW

File: GliaWidgets/Theme/Survey/Theme.Survey.OptionButton.swift [hunk 1]
```diff
@@ -39,5 +39,32 @@ extension Theme.SurveyStyle {
...
+            normalLayer.applyLayerConfiguration(optionButton?.normalLayer)
+            selectedText.applyTextConfiguration(optionButton?.selectedText)
+            selectedLayer.applyLayerConfiguration(optionButton?.selectedLayer)
+            highlightedText.applyTextConfiguration(optionButton?.highlightedText)
+            highlightedLayer.applyLayerConfiguration(optionButton?.highlightedLayer)
+        }
+
+        /// Apply option button title font from remote configuration
+        private mutating func applyFontConfiguration(_ font: RemoteConfiguration.Font?) {
+            if let fontSize = font?.size, let fontStyle = font?.style {
```
Comment (EgorovEI) [inline GliaWidgets/Theme/Survey/Theme.Survey.OptionButton.swift:56]: Will be better to move it to separate static method.
```suggestion
 extension UIColor {
    static func font(size: CGFloat, style: RemoteConfiguration.FontStyle) -> UIFont {
        switch fontStyle {
        case .bold:
            return UIFont.boldSystemFont(ofSize: fontSize)
        case .italic:
            return UIFont.italicSystemFont(ofSize: fontSize)
        case .regular:
            return UIFont.systemFont(ofSize: fontSize)
        case .thin:
            return UIFont.systemFont(ofSize: fontSize, weight: .thin)
        }
    }
}

if let fontSize = font?.size, let fontStyle = font?.style {
    self.font = UIFont(size: fontSize, style: fontStyle)
}

```

Comment (rasmustautsglia) [reply GliaWidgets/Theme/Survey/Theme.Survey.OptionButton.swift]: FIxed

File: GliaWidgets/Theme/Survey/Theme+Survey.swift [hunk 2]
```diff
@@ -215,4 +104,21 @@ extension UIFont {
         default:        return 0.2
         }
     }
+
+    static func convertToFont(font: RemoteConfiguration.Font?) -> UIFont? {
```
Comment (EgorovEI) [inline GliaWidgets/Theme/Survey/Theme+Survey.swift:108]: Will be better to move it to separate file. For example to `GliaWidgets/Lib/Extensions`

Comment (rasmustautsglia) [reply GliaWidgets/Theme/Survey/Theme+Survey.swift]: I moved the static function to a separate file.

**Review by EgorovEI — APPROVED** [review]

**Review by dukhovnyi — APPROVED** [review]

**Comment by rasmustautsglia** [comment]
!merge

**Pull request merged**